nengo_viz/static/viz_slider.js

```diff
@@ -151,6 +127,41 @@ VIZ.Slider = function(args) {
 VIZ.Slider.prototype = Object.create(VIZ.Component.prototype);
 VIZ.Slider.prototype.constructor = VIZ.Slider;
 
+VIZ.Slider.prototype.set_value = function(graph, slider, min_in, max_in, value) {
+    /** check if click was the middle mouse button */
```

Comment:
I'm not sure about min_in and max_in.  Those really seem like things the slider itself should know about, not things that have to be passed in to a call to ```set_value```.  Instead, I'd recommend adding ```max_value``` and ```min_value``` to the slider object.

I'd do this up around lines 26-28:
```
        slider.index = i;
        slider.div = document.createElement('div');
        slider.value = 0;
        slider.min_value = -1;    // add these lines
        slider.max_value = 1;    // add these lines
```